.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: python
 python: 3.6
 cache: pip
-before_install: pip install -U pre-commit
+before_install:
+- pip install -U pre-commit
+- pip install pylint-django==0.11.1
+- pip install -r requirements.txt
```

Comment:
Why did you split just one requirement into a file, while listing others outside of it? Be consistent.